hash_practice/exercises.py

```diff
@@ -2,18 +2,61 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(nlogn) - beacuse of the sort
+        Space Complexity: O(n)
     """

```

Comment:
👍 If the words are limited in length, like limited to english words which can only be so long it would be O(n) where n is the number of words.